marketplace/netlify-build/src/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { init } from 'contentful-ui-extensions-sdk';
+import { createPubSub } from './pubnub-client';
+import { Select, Option } from '@contentful/forma-36-react-components';
+import NeflifySideBarBuildButton from './NetlifySideBarBuildButton';
+
+function createSiteStructure(parameters = {}) {
+  const buildHooks = parameters.buildHookUrls.split(',');
```

Comment:
I'd always default to an empty string `(parameters.buildHookUrls || '').split(',')`

Reply:
Good call, will change.